### PR TITLE
fix: spacing

### LINKED ISSUE #104: Too much space between an image and columns block
There seem to be too much space between image and column block, original left, franklin site right:
image

### PR DESCRIPTION
**Pull request by cuzuco2**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #104

Test URLs:
- Before: https://main--elco-aerotop-sg--hlxsites.hlx.page/de/
- After: https://104-fix-space--elco-aerotop-sg--hlxsites.hlx.page/de/

### CODE REVIEW

File: styles/styles.css [hunk 1]
```diff
@@ -375,6 +375,11 @@ main {
 }
 
 main .section {
+  padding: 45px 30px 65px;
+  overflow: hidden;
```
Comment (solaris007) [inline styles/styles.css:381]: why is this needed?

**Review by solaris007 — DISMISSED** [review]

**Comment by ekremney** [comment]
i just noticed that some other parts in the dom (such as the one starts with "Leiser als in einer Bibliothek." has some top padding/margin discrepancy with the original site. Is possible if you include that space-fix into this PR as well?

File: styles/styles.css (hunk 1 above)
Comment (cuzuco2) [reply styles/styles.css:381]: padding: 45px 30px 65px;?
I guess that is the general padding of the main sections?
Or which one are you talking about?

Comment (solaris007) [reply styles/styles.css:381]: my comment refers to the `overflow: hidden`

Comment (cuzuco2) [reply styles/styles.css:381]: I didn't put that one but I see it is used on the original site, probably for lower screen widths to avoid having scrollbars:
image

**Review by solaris007 — APPROVED** [review]
